src/app/adapters/fallbackAdapter.js

```diff
@@ -27,15 +27,20 @@ export class FallbackStaticAdapter implements StaticPluginAdapter {
 
   nodeTypes() {
     return [
-      {name: "(unknown node)", prefix: NodeAddress.empty, defaultWeight: 1},
+      {
+        name: "unknown node",
+        pluralName: "unknown nodes",
+        prefix: NodeAddress.empty,
+        defaultWeight: 1,
+      },
     ];
   }
 
   edgeTypes() {
     return [
       {
-        forwardName: "(unknown edge→)",
-        backwardName: "(unknown edge←)",
+        forwardName: "unknown edge→",
```

Comment:
FWIW, I think that these names are confusing. How is a user supposed to
know what `edge→` means? Something like “unknown out-edge” would be
clearer—but no change necessary for this PR.